docs/examples/sglang/README.md

```diff
@@ -0,0 +1,100 @@
+# Deploy Distributed Inference Service with SGLang and LWS on GPUs
+
+In this example, we demonstrate how to deploy a distributed inference service using LeaderWorkerSet (LWS) with [SGLang](https://docs.sglang.ai/) on GPU clusters.
+SGLang provides native support for distributed tensor-parallel inference and serving, enabling efficient deployment of large language models (LLMs) such as DeepSeek-R1-671B and Meta-Llama-3.1-405B across multiple nodes. This example uses the [meta-llama/Meta-Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct) model to demonstrate multi-node serving capabilities. For implementation details on distributed execution, see the SGLang docs [Run Multi-Node Inference](https://docs.sglang.ai/references/multi_node.html).
+
```

Comment:
Also add a note here:
```
Since SGLang employs tensor parallelism for multi-node inference, 
which requires more frequent communications than pipeline parallelism, 
ensure high-speed bandwidth between nodes to avoid poor performance.
```